example/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: "2.4"
+services:
+  borgmatic:
+    container_name: borgmatic
+
+    image: modem7/borgmatic-docker
```

Comment:
Should this be the docker-borgmatic ghcr.io container uri?

Reply:
Didn't even realize that existed :V I based this off [the link](https://www.modem7.com/books/docker-backup/page/backup-docker-using-borgmatic) at the bottom of [the README](https://github.com/borgmatic-collective/docker-borgmatic/blob/master/README.md) - it actually took a bit of work to find the ghcr.io container, even knowing it existed.

I've updated this!

Reply:
My local install, changed to this image, still works fine with a manual backup, I'll tell you tomorrow if the cron backup still works (though I have no reason to believe it won't)

Reply:
Seems to work, though it's a lot spammier and harder to read the log :V So it goes though.